spec/controllers/hyrax/collections_controller_spec.rb

```diff
@@ -78,7 +79,9 @@
         end
 
         it "sets breadcrumbs" do
-          expect(controller).to receive(:add_breadcrumb).with('My Dashboard', Hyrax::Engine.routes.url_helpers.dashboard_path(locale: 'en'))
+          expect(controller).to
```

Comment:
You can't have a line break here.  The ruby interpreter doesn't understand how to parse this correctly.

try this instead: 
```ruby
expect(controller).to receive(:add_breadcrumb)
  .with(...)
```